pkg/db/file_impl.go

```diff
@@ -56,4 +59,118 @@ func (F *FSBackend) Files() ([]string, error) {
 	return names, nil
 }
 
+//used for storing things in a file (remove if DB starts being used)
+//create a category-json file in the OS's temp directory
+var ex, _ = os.Executable()
+var exPath = filepath.Dir(ex)
+var categoriesFile = exPath + "/cat.json"
```

Comment:
Let's move this to the struct. IE, the .json file can just live in the dir used for storing the files. I do not expect collisions with this implementation, and it's a little bit neater than trying to write relative to the execution dir (which won't always be where the source lives)

Reply:
ah, didn't even notice dirLocation on FSBackend